bootstrap.js

```diff
@@ -370,6 +370,9 @@ windowObserver = {
       for (let item of menu.querySelectorAll(".newtabtools-item")) {
```

Comment:
You could just change this string to `".newtabtools-item, .newtabtools-page"`. I'd rather it was done that way.